lang/build.gradle

```diff
@@ -62,16 +62,70 @@ tasks.register('javadocJar', Jar) {
 
 publishing {
     publications {
-        create('maven', MavenPublication) {
-            artifactId = 'partiql-lang-kotlin'
+        maven(MavenPublication) {
+            // group id and version are set in the root build.gradle for all projects.
+            artifactId = "partiql-lang-kotlin"
+
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
 
-            from(components['java'])
-            artifact(tasks['sourcesJar'])
-            artifact(tasks['javadocJar'])
+            pom {
+                name = "PartiQL-Lang-Kotlin"
+                packaging = "jar"
+                url = "https://github.com/amzn/ion-schema-kotlin"
```

Comment:
wrong url, should be https://partiql.org/

Reply:
oops--removed in next commit.